extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -36,8 +39,135 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
   transition: opacity 0.2s cubic-bezier(0, 0, 0.2, 1) !important;
 }
 
-/** Make the dialog wrapper from amp-subscriptions round corner. */
+/** Override the dialog styles from amp-subscriptions. */
 amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
-  border-radius: 16px 16px 0px 0px !important;
+  border-radius: 0.9em 0.9em 0 0 !important;
   font-family: Poppins, sans-serif !important;
+  font-weight: 400 !important;
+  background: white !important;
+  color: black !important;
+  overflow: hidden !important;
+}
+
+amp-story:not([supports-landscape]) + amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
+  width: 100% !important;
+  left: 0 !important;
+  margin-left: 0 !important;
+}
+
+@media(min-aspect-ratio: 3/4) {
+  amp-story:not([supports-landscape]) + amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
+    width: var(--i-amphtml-story-desktop-one-panel-width) !important;
+    left: 50% !important;
+    margin-left: calc(-0.5 * var(--i-amphtml-story-desktop-one-panel-width)) !important;
+  }
+}
+
+.i-amphtml-story-subscriptions-dialog-banner {
+  height: 2.54em !important;
+  background: #EDEDED !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-content {
+  margin: 2.1em 1.6em 0 !important;
+  text-align: center !important;
+  white-space: nowrap !important;
+}
+
+.i-amphtml-story-subscriptions-price {
+  display: block !important;
+  font-weight: 400 !important;
+  font-size: 1.4em !important;
+  letter-spacing: -0.02em !important;
+  text-overflow: ellipsis !important;
+  overflow: hidden !important;
+  margin-bottom: 0.2em !important;
+}
+
+.i-amphtml-story-subscriptions-title {
+  display: block !important;
+  font-weight: 700 !important;
+  font-size: 1.15em !important;
+  letter-spacing: -0.015em !important;
+  text-overflow: ellipsis !important;
+  overflow: hidden !important;
+  margin: 0.4em 0 0.15em !important;
+}
+
+.i-amphtml-story-subscriptions-subtitle-first, .i-amphtml-story-subscriptions-subtitle-second {
+  display: block !important;
+  font-size: 0.9em !important;
+  letter-spacing: -0.015em !important;
+  text-overflow: ellipsis !important;
+  overflow: hidden !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-button, .i-amphtml-story-subscriptions-google-button {
+  margin: 0 auto 0.8em !important;
+  padding: 0 !important;
+  border: none !important;
+  width: 12.2em !important;
```

Comment:
We need to be sure the button is responsive to its content. It needs to shrink and grow if it has less or more.
<img width="230" alt="Screen Shot 2022-04-14 at 12 14 02 PM" src="https://user-images.githubusercontent.com/3860311/163441085-66cad5fe-8f5a-4fcf-b044-bf0e027f5569.png">

Do do this we need to: 
1. Remove the hard coded width here.
2. Add `margin-inline-end: 1.3em;` to the text child for responsive/RTL margin.
3. `display: flex; flex-direction: column;` Need to be on the parent, in this case `i-amphtml-story-subscriptions-dialog-content` this will make the elements shrink to content and be centered.